lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -3035,3 +3046,64 @@ instance HasSeverityAnnotation WalletEngineLog where
     getSeverityAnnotation = \case
         MsgWallet msg -> getSeverityAnnotation msg
         MsgSubmitSealedTx msg -> getSeverityAnnotation msg
+
+forgeToken
+    :: forall ctx s k n.
+        ( ctx ~ ApiLayer s k
+        , Bounded (Index (AddressIndexDerivationType k) 'AddressK)
+        , GenChange s
+        , HardDerivation k
+        , HasNetworkLayer ctx
+        , IsOwned s k
+        , Typeable n
+        , Typeable s
+        , WalletKey k
+        )
+    => ctx
+    -> ArgGenChange s
+    -> ApiT WalletId
+    -> Api.ForgeTokenData n
+    -> Handler (ApiTransaction n)
+forgeToken ctx genChange (ApiT wid) body = do
+    let pwd = coerce $ body ^. #passphrase . #getApiT
+    let forgePayments = fmap (\(AddressForgeAmount (ApiT addr, _) amt) -> AddressForgeAmount addr amt) $ body ^. #forgePayments
```

Comment:
Suggestion: use `<&>` here to break up this long line.

```suggestion
    let forgePayments = body ^. #forgePayments <&>
            \(AddressForgeAmount (ApiT addr, _) amt) ->
                AddressForgeAmount addr amt
```

Another alternative would be to define a named function, and use that with plain vanilla `<$>`.